features/projects/src/main/kotlin/com/agilefreaks/freaks_catalog/features/projects/details/ProjectDetailsFragmentModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.agilefreaks.freaks_catalog.features.projects.details
+
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val detailsFragmentModule = module {
```

Comment:
It's not only here the problem, but this variable and this file name should'n contain fragment in it. It doesn't have anything to do with fragments